OSPSuite.Presentation.Importer/packages.config

```diff
@@ -0,0 +1,9 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="LumenWorksCsvReader" version="4.0.0" targetFramework="net472" />
+  <package id="NPOI" version="2.5.1" targetFramework="net472" />
+  <package id="OSPSuite.DevExpress" version="19.2.4.1" targetFramework="net472" />
+  <package id="OSPSuite.Utility" version="4.0.0.4" targetFramework="net472" />
+  <package id="Portable.BouncyCastle" version="1.8.6" targetFramework="net472" />
```

Comment:
What is this?

Reply:
BouncyCastle is used by npoi. Without it npoi does not run.